Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs

```diff
@@ -175,35 +185,83 @@ protected static IdentityDbContext IdentityDbContext
             }
         }
 
-        public HttpClient HttpClient { get; }
+        private readonly TaskCompletionSource<HttpClient> _httpClientTaskCompletionSource;
+
+        public Task<HttpClient> HttpClient => _httpClientTaskCompletionSource.Task;
 
         private TestServer CreateIdentityTestServer(string storageProvider)
         {

```

Comment:
Can this test server setup be consolidated some, and create the TestServer with a method and only the custom settings for each test server should be different.

Reply:
Thanks for the input, I fixed this and will push the changes soon.